test/infrastructure/vm-operator/vm-operator-console-patch.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: apps/v1
```

Comment:
This file is unused?!

Reply:
yes, we are dropping this Deployment entirely now